releasenotes/notes/integrated-quantum-bayesian-inference-92c6025432d9b7e0.yaml

```
@@ -0,0 +1,17 @@
+---
+features:
+  - |
+    Introduction of the `QBayesian` class in the Qiskit Machine Learning library.
```

```suggestion
    Introduction of the :class:`.QBayesian` class in the Qiskit Machine Learning library.
```
if at least one of these refs is made a Sphinx class ref that will end up as a link to the class in the API ref so and end user can simply click in the release notes on that link to go to it. In this case by doing . there Sphinx will resolve it to where it actually exists.

---

BTW is you want to see how the docs come out in the CI build here click on one of the Details in the list below and then navigate to Summary. If you scroll down the you will find a number of artifacts that are built and can be downloaded. `documentation.zip` will have the docs with the tutorials just as they are checked-in. `tutorials3.8` and the 3.11 one will have the docs plus the tutorials are run in the build under Python 3.8 (or 3.11) so the outputs are updated from what is checked in and we can make sure they still run.

---

Makes sense. I will do so.